### PR TITLE
Add lastest Page and comment backend api

### PR DESCRIPTION
**Pull request by luomi16**
# Description

Add latest page based on published date

### CODE REVIEW

**Comment by Hillary764** [comment]
It looks like there's a couple issues with prettier / eslint rules.  Can you go back and fix them or should I go back? It looks like it's just formatting.

Sorry, haven't given a detailed look yet, was getting the preview set up

**Comment by luomi16** [comment]
Sure, I will check it out as soon as possible and fix it! Thank you for
your reply.

Best,
Mi Luo

File: src/app/stories/latest/page.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    : "ALL TOPICS | LATEST";
+
+  return (
+    <>
+      <div className="mx-[10%] my-10 flex flex-col gap-12">
+        <HomepageSection heading={headerText}>
+          {stories.length > 0 ? (
+            <>
+              <ArticleList articles={stories} preferHorizontal={true} />
+              <Pagination total_pages={total_pages} />
```
Comment (Hillary764) [inline src/app/stories/latest/page.tsx:36]: I also feel like the black text on the pagination numbers might be better in white, it's a little difficult to see on the relatively dark background

File: src/components/Header/index.tsx [hunk 2]
```diff
@@ -125,7 +126,7 @@ export default function Header() {
           className=" grow cursor-pointer py-2 transition-colors hover:bg-sciquelHover"
           style={{ display: scroll ? "none" : "block" }}
         >
-          <Link href="/">LATEST</Link>
+          <Link href="/stories/latest">LATEST</Link>
```
Comment (Hillary764) [inline src/components/Header/index.tsx:129]: nice, I didn't notice that needed an update.  Should we update the home page to also reflect this new page? (change the "read all recent" link? I don't know if it matters

File: prisma/schema.prisma [hunk 3]
```diff
@@ -283,9 +285,27 @@ model UserResponse {
...
 
 model ResponseSubpart {
-  id              String        @id @default(auto()) @map("_id") @db.ObjectId
-  subpartId       String        @db.ObjectId
-  subpartUserAns  String[]
-  UserResponse    UserResponse? @relation(fields: [userResponseId], references: [id])
-  userResponseId  String?       @db.ObjectId
+  id             String        @id @default(auto()) @map("_id") @db.ObjectId
+  subpartId      String        @db.ObjectId
+  subpartUserAns String[]
```
Comment (Hillary764) [inline prisma/schema.prisma:290]: can I ask what subpartUserAns is used for?

**Review by Hillary764 — COMMENTED** [review]
I left comments on the files, this looks mostly good, but there are a couple clarifying questions and minor edits I had

**Comment by luomi16** [comment]
Thank you for your useful feedback!! These means a lot to me!

File: prisma/schema.prisma (hunk 3 above)
Comment (luomi16) [reply prisma/schema.prisma:290]: This isn't what I've done; the code formatter added some space.

File: src/components/Header/index.tsx (hunk 2 above)
Comment (luomi16) [reply src/components/Header/index.tsx:129]: To my understanding, this is the entry from the latest page. I'm not sure if it's correct.

File: src/app/stories/latest/page.tsx (hunk 1 above)
Comment (luomi16) [reply src/app/stories/latest/page.tsx:36]: Yes, that might be better. We could make the modification in components/StoriesList/PaginationNumber.tsx. 
(I also write details for my own reference. :))

File: src/app/api/stories/latest/route.ts
[the remark this answers is not in the document]
Comment (luomi16) [reply src/app/api/stories/latest/route.ts:1]: Yes, I think the method you mentioned is better. I tried to implement it at the beginning, but I didn't figure out, so I ended up using a similar template to api/stories.

File: src/app/api/comments/route.ts [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+    }
+
+    const commentData = parsed.data;
+
+    const createdComment = await prisma.comment.create({
+      data: {
+        content: commentData.content,
+        quote: commentData.quote,
+        parentCommentId: commentData.parentCommentId,
+        userId: commentData.userId,
```
[the remark this answers is not in the document]
Comment (luomi16) [reply src/app/api/comments/route.ts:32]: Okay, Hillary, thank you for your reminder.

**Pull request closed without merging**